### PR TITLE
chore: add validator for dos protec metrics and move to app

### PR DESCRIPTION
**Pull request by alrevuelta**
- [x] Add metrics for signed message validator [see](https://rfc.vac.dev/spec/57/#dos-protection) indicating amount of accepted and rejected messages.
- [x] Move said validator to app
- [x] Move tests to wakunode2 app tests.

Related: https://github.com/waku-org/nwaku/issues/1612

### CODE REVIEW

**Comment by alrevuelta** [comment]
Tests are failing, addressing the changes.

File: tests/wakunode2/test_validators.nim [hunk 1]
```diff
@@ -0,0 +1,265 @@
+{.used.}
```
Comment (alrevuelta) [inline tests/wakunode2/test_validators.nim:1]: just a copy paste of the tests, nothing changed

**Comment by alrevuelta** [comment]
Fixed!

**Review by rymnc — APPROVED** [review]
LGTM!

**Review by jm-clius — APPROVED** [review]
LGTM, thanks!

**Pull request merged**